src/ert/_c_wrappers/enkf/config/field_config.py

```diff
@@ -108,6 +113,67 @@ def ijk_active(self, i, j, k):
     def free(self):
         self._free()
 
+    def truncate(self, data: npt.ArrayLike) -> npt.ArrayLike:
+        # Truncation is not mentioned in the docs
+        # Its really min / max
+
+        truncation_mode = self._get_truncation_mode()
+        if truncation_mode == 0:
+            return data
+        if truncation_mode == 1:
+            min_ = self.get_truncation_min()
+            vfunc = np.vectorize(lambda x: max(x, min_))
+            return vfunc(data)
+        if truncation_mode == 2:
+            max_ = self.get_truncation_max()
+            vfunc = np.vectorize(lambda x: min(x, max_))
+            return vfunc(data)
+        if truncation_mode == 3:
+            min_ = self.get_truncation_min()
+            max_ = self.get_truncation_max()
+            vfunc = np.vectorize(lambda x: max(min(x, max_), min_))
+            return vfunc(data)
+
+    def transform(self, transform_name: str, data: npt.ArrayLike) -> npt.ArrayLike:
+
+        if not transform_name:
+            return data
+
+        func = None
+        if transform_name == "LN" or transform_name == "LOG":
+            func = lambda x: math.log(x, math.e)
+        elif transform_name == "LN0":
+            func = lambda x: math.log(x, math.e) + 0.000001
+        elif transform_name == "LOG10":
+            func = lambda x: math.log(x, 10)
+        elif transform_name == "EXP":
+            func = lambda x: math.exp(x)
+        elif transform_name == "EXP0":
+            func = lambda x: math.exp(x) + 0.000001
+        elif transform_name == "POW10":
+            func = lambda x: math.pow(x, 10)
+        elif transform_name == "TRUNC_POW10":
+            func = lambda x: math.pow(max(x, 0.001), 10)
+        elif transform_name == "DENORMALIZE_PERMX":  # not in docs
+            pass
+        elif transform_name == "DENORMALIZE_PORO":  # not in docs
+            pass
+        elif transform_name == "DENORMALIZE_PERMZ":  # not in docs
+            pass
+        elif transform_name == "NORMALIZE_PORO":  # not in docs
+            pass
+        elif transform_name == "NORMALIZE_PERMZ":  # not in docs
+            pass
+        elif transform_name == "NORMALIZE_PERMX":  # not in docs
+            pass
+
+        if not func:
```

Comment:
Is this the old behavior? Sounds like maybe we should validate this in the configuration and raise an exception before ert starts instead? Not sure how much work that is, but then the whole  `pass` section can be omitted.

Reply:
Agree that this should be caught in the validation. I left the statements here for discussion. Will remove before final merge